test/end-to-end/pages/createImage.js

```diff
@@ -20,7 +20,7 @@ module.exports = class CreateImagePage extends MainPage {
     this.labelCreateImage = `${this.dialogRootElement} #myModalLabel`;
 
     // Blueprint Name label
-    this.labelBlueprintName = `${this.dialogRootElement} p[class="form-control-static"]`;
+    this.labelBlueprintName = `${this.dialogRootElement} #cmpsr-form-blueprint`;
```

Comment:
Related to the previous comment, since we are using `data-id` instead of `id`, then the attribute selector should be updated to:
```
this.labelBlueprintName = `${this.dialogRootElement} [data-id="cmpsr-form-blueprint"]`;
```